src/client/opamCommands.ml

```diff
@@ -1480,9 +1480,15 @@ let install =
        installed. If not, output the names of the missing dependencies to \
        stdout, and exits with 1."
   in
+  let depext_only =
+    mk_flag ["depext-only"]
+      "Install external dependencies required by $(i,PACKAGES) and their \
+       dependencies, and quit"
```

Comment:
Is it worth also forbidding `--deps-only --depext-only` with an error?

Reply:
It is actually handled, both option can be passed. Why do you think it should be forbidden?

Reply:
Ah I see, so `--depext-only --deps-only` will install just the depexts for the dependencies, and not the main package?  It will also not install the opam packages for the dependencies.

That's worth clarifying in the help message if that's the case...

Reply:
Yes, in that case it only installs dependencies depexts. I'll update the doc.